modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -35,5 +35,23 @@
 		],
 		"newReference": "HtmlParser _htmlParser",
 		"to": "_htmlParser.extractText"
+	},
+	{
+		"classNames": [
+			"DLFolderLocalService",
+			"DLFolderLocalServiceUtil"
+		],
+		"from": "addFolder(long, long, long, boolean, long, String, String, boolean, ServiceContext)",
+		"issueKey": "LPS-199538",
+		"to": "addFolder(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#7#, param#8#)"
+	},
+	{
+		"classNames": [
+			"DLFolderService",
+			"DLFolderServiceUtil"
+		],
+		"from": "addFolder(long, long, boolean, long, String, String, ServiceContext)",
+		"issueKey": "LPS-199538",
+		"to": "addFolder(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#)"
 	}
```

Comment:
@MicaelleSilvaa please sort this, this pattern should come first over the other.

Reply:
resolved